lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -5,6 +5,7 @@
     "copy-from-recipe": {
         "etc/": "%ETC_DIR%"
     },
+    "aliases": ["jwt-auth"],
```

Comment:
I wanted at first, but the bundle is a solution for setting a jwt authentication, not for consuming external jwt tokens for instance (even if it requires libraries that will probably do natively).